Makefile

```diff
@@ -203,9 +203,44 @@ check-image: build-tui
 	fi
 	@echo "consider running 'sudo make clean' to remove the files built as root"
 
+PHONY += check-all-images-core
+check-all-images-core:
+	@if [ "${runuid}" != "0" ] ; then \
+		echo "check-image needs to be run as root; please use 'sudo'"; \
+		/bin/false; \
+	fi
+
+PHONY += check-all-images
+check-all-images: check-all-images-core
+	@echo "This process runs serially and take a long time and should not be aborted!"
+	@echo "Consider instead 'make check-all-images-screen'"
+	@echo "Waiting 10 seconds to start..."
+	@sleep 10
+	@cd ${top_srcdir} ; tests/check-all-images.sh .
+	@echo "consider running 'sudo make clean' to remove the files built as root"
+
+PHONY += check-all-images-screen
+check-all-images-screen: check-all-images-core
+	@echo "Lauching screen sessions..."; \
+	echo "Use <control>-a<controld>-d to detach and leave running."; \
+	echo "	sudo screen -R # to recover session"
+	@sleep 2
+	@screen_log="screen-check-all-images.log"; \
+	if [ -f $${screen_log} ] ; then \
+		mv -f $${screen_log} $${screen_log}.save; \
+	fi; \
+	cd ${top_srcdir} ; screen -L -Logfile $${screen_log} tests/check-all-images.sh . ; \
```

Comment:
Maybe the change directory into the top_srcdir is redundant as a user needs already needs to exist there to invoke make target